frontend/public/components/catalog/catalog-items.tsx

```diff
@@ -303,8 +300,8 @@ export class CatalogTileViewPage extends React.Component<
             <Badge isRead>{filter.label}</Badge>
           </CatalogTileBadge>,
         ]}
-        iconImg={iconImgUrl}
-        iconClass={iconClass}
+        iconImg={tileImgUrl}
+        iconClass={tileIconClass}
```

Comment:
You've removed the call to `normalizeIconClass` along with removing the default catalog image as a fallback.
Couldn't you simply omit the default image if there is a class provided?
```suggestion
        iconImg={tileImgUrl || (iconClass ? undefined : catalogImg)}
        iconClass={iconClass}
```